### ISSUE TITLE
An issue in loading .gltf files

### ISSUE THREAD

**Issue by deep9888** [KhronosGroup/gltf-asset-auditor#6]
An issue in loading .gltf files
I am facing an issue with loading .gltf files. Two hours before, it worked well, but now it gives me an error of -
```
The total Size is 10749640
ERROR: Error loading model: Offset is outside the bounds of the DataView
```
Following are the files in which folder contains-
![ERROR4](https://user-images.githubusercontent.com/103727216/231167080-c42fb3df-6725-4bbc-bbcb-fd294736d547.png)

Can someone guide me to where the issue is?

**Comment by MikeFesta** [comment]
It's strange that it was working earlier and then stopped. Perhaps not all of the files were selected when you got that error. I've seen a similar message when a file that the .gltf tried to reference was not found.

As a next step, I'd recommend trying to load the model in the glTF Validator: https://github.khronos.org/glTF-Validator/
If that works, then try the Babylon sandbox: https://sandbox.babylonjs.com/

**Comment by deep9888** [comment]
I also load the model in the glTF Validator: https://github.khronos.org/glTF-Validator/. It gives me the asset is valid, and the same model I load with three js works fine. According to me, the issue is in **Glb.ts which contains TotalSize and TotalSize is less than ArrayBuffer Size**.

**Comment by deep9888** [comment]
I also tried in Babylon sandbox: https://sandbox.babylonjs.com/, model is working.
![error2](https://user-images.githubusercontent.com/103727216/230917053-cfaeff91-1d5d-4913-b2be-b11e9d5e7c83.png)

**Comment by MikeFesta** [comment]
Did anything change with the images between when it was working and when it stopped working? Were they edited or changed? Do you know if they are 8-bits per channel or 16-bits per channel? RGBA or Grayscale?

**Comment by deep9888** [comment]
I didn't change anything from my side. The images are of RGBA,  I tried with 2 to 3 .gltf files for testing, but with all, it gives me the same error.
![Screenshot 2023-04-10 195142](https://user-images.githubusercontent.com/103727216/231169659-926f0801-e088-4baa-ae19-837c108a2d48.png)

**Comment by MikeFesta** [comment]
You mentioned that it was working well two hours before. Was that with the same gltf and files? Nothing changed in the repo, so I'm not sure what would have caused it to break for you.

**Comment by deep9888** [comment]
Yes, it was working well. And suddenly, it stopped working. I didn't change anything in the code, and also I tested 2 to 3 gltf files on your web-deployed platform GLB files are working, but the gltf files are not working. I double-checked my code; it is working as expected, and the gltf files are valid.

**Comment by deep9888** [comment]
I also tried with another gltf file, and it gave me the same error-
![error3](https://user-images.githubusercontent.com/103727216/231073369-d0bd76d2-f7bd-4d91-a697-d1d6e613cc9e.png)

**Comment by dukemagus** [comment]
check if this PR fixed it.

https://github.com/KhronosGroup/gltf-asset-auditor/pull/7

if this solves the problem, please close the issue
